tests/unit_tests/lcls_tools/superconducting/test_sc_cryomodule.py

```diff
@@ -22,5 +22,5 @@ def test_pv_prefix(self):
         self.assertEqual(cm.pv_prefix, "ACCL:L0B:0100:")
 
     def test_num_cavities(self):
-        cm = MACHINE.cryomodules[f"{randint(1, 35):02d}"]
+        cm = MACHINE.cryomodules[f"{randint(1, 35): 02d}"]
```

Comment:
Sorry, there's one more down the line I didn't point out. I think the test will pass after this change.